react/FilePicker/Readme.md

```diff
@@ -41,7 +38,7 @@ const onChange = (fileId) => alert(`ID of file selected : [${fileId}]`);
             <FilePicker
               onClose={toggleFilePicker}
               onChange={onChange}
-              accept={acceptRule}
+              accept={acceptRule.toString()}
```

Comment:
I don't see any `itemsIdsSelected` example in README inside this PR. Don't we want to document that new feature?

Reply:
`itemsIdsSelected` is an internal state of the FilePicker.
It corresponds to the value returned by the `onChange` function